src/Workspaces/Remote/Core/ExternalAccess/UnitTesting/Api/UnitTestingRemoteHostClient.cs

```diff
@@ -33,6 +33,7 @@ internal UnitTestingRemoteHostClient(ServiceHubRemoteHostClient client, UnitTest
             return new UnitTestingRemoteHostClient((ServiceHubRemoteHostClient)client, serviceDescriptors, callbackDispatchers);
         }
 
+        [Obsolete("Use UnitTestingGlobalOptions.IsServiceHubProcessCoreClr instead")]
         public static bool IsServiceHubProcessCoreClr(HostWorkspaceServices services)
         {
             var optionServices = services.GetRequiredService<IOptionService>();
```

Comment:
This is problematic since we do not want global options to be accessed from remote workspace layer and potentially oop where they are not available.